client/css/main.css

```diff
@@ -0,0 +1,151 @@
+html { 
+  width : 100%;
+  height : 100%;
+}
+
+body { /*setting background image flexible to the screen with background colour for a case of fall over*/
+  background-color: rgba(162, 176, 255, 1);
+  background-image:  linear-gradient(#101e49c7,  #495bf542), url("../image/bg.PNG") !important;
```

Comment:
Is `!important` needed here?

Reply:
Originally It was there because background colour took the screen.
Somehow it is okay now so I removed it from all css files.